src/components/core/check-overflow/index.js

```diff
@@ -1,9 +1,16 @@
 
 function checkOverflow() {
   const swiper = this;
+  const params = swiper.params;
   const wasLocked = swiper.isLocked;
+  const lastSlidePosition = (params.slidesOffsetBefore + (params.spaceBetween * (swiper.slides.length - 1)) + ((swiper.slides[0]).offsetWidth) * swiper.slides.length);
```

Comment:
When passing no slides this would throw: `Uncaught TypeError: Cannot read property 'offsetWidth' of undefined`
Probably a good idea to conditional check this slides before setting lastSlidePosition.

Reply:
Ah, good catch. Added a fix for that now.